rfcs/20251105-rms-norm-graph-op/README.md

```diff
@@ -0,0 +1,321 @@
+# Support Root Mean Square Normalization in Graph API
+=====================================================================
+
+## Motivation
+
+Root Mean Square Normalization (RMSNorm)[[#1]][1] has emerged as a critical component in
+modern large language models (LLMs) and transformer architectures. RMSNorm
+provides a simpler and more efficient alternative to Layer Normalization by
```

Comment:
Compare to Layer Normalization, RMSNorm is more efficient in normalizing inputs by using only the root ...

Reply:
fixed. thanks a lot